static/styles/subscriptions.scss

```diff
@@ -713,18 +701,42 @@ form#add_new_subscription {
     margin-right: 5px;
 }
 
-.stream-row .sub-info-box .bottom-bar > div {
+.stream-row .settings-dropdown-trigger {
+    float: right;
+    margin-left: 10px;
+    color: hsl(0, 0%, 67%);
+}
+
+.stream-row.active {
+    background-color: hsl(0, 0%, 93%);
+}
+
+.stream-row > div {
     display: inline-block;
-    vertical-align: bottom;
+    vertical-align: top;
 }
 
-.stream-row .sub-info-box .bottom-bar {
-    margin-top: 2px;
-    line-height: 1.5;
+.stream-row:hover .preview-stream {
+    display: inline-block;
 }
 
-#subscription_overlay .stream-description .stream-description-editable:empty::after,
```

Comment:
This line seems to have been split with a comma?  How did you generate this commit ?  I would have imagined one would have done it with just copy-pasting rearranging things to avoid risk of introducing unexpected changes like this.

```
#subscription_overlay .stream-description,
￼	.stream-description-editable:empty::after {
￼```	

If you're doing more complex changes to some of the blocks to re-split them or something, please do that in a separate commit from simple reordering -- it's really unpleasant to verify a reordering commit squashed with other changes.